xchange-core/src/main/java/org/knowm/xchange/dto/account/FundsRecord.java

```diff
@@ -0,0 +1,79 @@
+package org.knowm.xchange.dto.account;
+
+import java.math.BigDecimal;
+
+public final class FundsRecord {
+  private final String address;
+  private final Long date;
+  private final String ccy;
+  private final BigDecimal amount;
+  private final String ref;
+  private final String fundsType;
```

Comment:
This should be an `enum`, eg. `public enum Type { WITHDRAWAL, DEPOSIT }` within the the FundsRecord class.